server/elastic/src/adapter.ts

```diff
@@ -215,10 +216,7 @@ class ElasticAdapter implements FullTextAdapter {
         filter: [
           {
             bool: {
-              should: [
-                { terms: this.getTerms(_classes, '_class') },
-                { terms: this.getTerms(_classes, 'attachedToClass') }
-              ]
+              must: [{ terms: this.getTerms(_classes, '_class') }]
```

Comment:
Why?

Reply:
Because we do not need to look on child, current indexer back propogate indexed fields to it's parent.